### PR TITLE
Removed vllm.hpu directory and changed relevant imports

### PR DESCRIPTION
**Pull request by tzielinski-habana**
Moved files from vllm/hpu to another public repo: https://github.com/HabanaAI/vllm-hpu-extension
It can be installed with pip install git+https://github.com/HabanaAI/vllm-hpu-extension.git

### CODE REVIEW

File: requirements-hpu.txt [hunk 1]
```diff
@@ -6,3 +6,4 @@ ray == 2.32.0
 triton
 pandas
 tabulate
+git+https://github.com/HabanaAI/vllm-hpu-extension.git
```
Comment (kzawora-intel) [inline requirements-hpu.txt:9]: can we fix that to a specific git hash?

**Review by kzawora-intel — CHANGES_REQUESTED** [review]
vllm.hpu needs to be removed from format.sh

File: requirements-hpu.txt (hunk 1 above)
Comment (kzawora-intel) [reply requirements-hpu.txt:9]:
```suggestion
git+https://github.com/HabanaAI/vllm-hpu-extension@e7e0579
```

Comment (tzielinski-habana) [reply requirements-hpu.txt]: Done, I fixed it to a newer version: https://github.com/HabanaAI/vllm-hpu-extension/commit/30ee2d1

**Comment by kzawora-intel** [comment]
you'll also need to change github action workflows similarly to format.sh: https://github.com/HabanaAI/vllm-fork/blob/habana_main/.github/workflows/mypy.yaml#L53

**Pull request merged**